CONTRIBUTING.md

```diff
@@ -49,6 +49,13 @@ See the readmes in the specific example app folders for individual instructions:
 - [ionic-angular-v2](example/ionic-angular-v2/README.md)
 - [ionic-angular-v3](example/ionic-angular/README.md)
 
+### Spotlight
+
+in order to test Spotlight, add the following comment to the .env file of each sample app that you wish to test:
+`export SENTRY_SPOTLIGHT_URL = http://IP:8969/stream;` where IP is your private IP.
```

Comment:
```suggestion
In order to test Spotlight, modify the file environment.local.ts or local.ts from the sample apps with the following value:
``typescript
export const localConfig = {
  spotlightSidecarUrl: 'http://IP:8969/stream' // replace IP by your local IP.
};
``

If the file is not present on your sample app, it will be automatically generated on the first build from the sample app.
```